.github/workflows/php7.2.yml

```diff
@@ -0,0 +1,98 @@
+name: "Website build"
+
+on: ["push", "pull_request"]
+
+env:
+  PHP_VERSION: "7.2"
+
+jobs:
+  tests:
+
+    runs-on: "ubuntu-16.04"
```

Comment:
Any specific reason to pick this version?

Reply:
Yes, it's the one that is used on the current server, that is involved on the deployment build.

Reply:
Lately I came to the idea that Github Actions might make this `ubuntu-16.04` server obsolete and we might be able to build the website on Actions alone. This would also make a planned PHP 7.4 update easier. Until then I prefer to keep this version for the time being.